packages/figma/src/codegen/targets/react/component/handlers/tabs.ts

```
@@ -1,131 +1,14 @@
+const TABS_UNDERLINE_FILL_ITEM_KEY = "78039b6b86852b685a6e93e9b6743b9e577cd7db";
 
-      const commonProps = {
-        value: props["Label#4478:2"].value,
-        ...(props.Notification.value === "True" && {
-          alert: true,
-        }),
-        ...(states.includes("Disabled") && {
-          disabled: true,
-        }),
-      };
+const { createLocalSnippetElement: createChipTabsLocalSnippetElement } =
```

Biome (2.1.2)

[error] This variable createChipTabsLocalSnippetElement is unused.

Unused variables are often the result of an incomplete refactoring, typos, or other sources of bugs.

(lint/correctness/noUnusedVariables)